### PR TITLE
style: addon size should inherit from props

### LINKED ISSUE #43993: addOn font size should be 16 when control size is large
### What problem does this feature solve?

image

Inconsistency between using text or component for addon, as component will inherit the size set in the Form, causing it to show fontSize 16, whereas addon text stays at 14

### What does the proposed API look like?

```
.ant-input-group-wrapper-lg .ant-input-group-addon {
  font-size: 16px;
}

.ant-input-number-lg   .ant-input-number-group-addon {
  font-size: 16px;
}
```

### PR DESCRIPTION
**Pull request by MuxinFeng**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/43993

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      addon size should inherit from props     |
| 🇨🇳 Chinese |     addon 的尺寸应该要从props继承      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 7eea7c1</samp>

This pull request enhances the input and input number components to support different font sizes based on the `size` prop. It updates the style files `components/input/style/index.ts` and `components/input-number/style/index.ts` to use the theme size tokens.

### <samp>🤖 Generated by Copilot at 7eea7c1</samp>

* Set the font size of input number and input group components according to the size prop ([link](https://github.com/ant-design/ant-design/pull/44000/files?diff=unified&w=0#diff-63d0bbc4ed8cb1745da77de552d75aa0fd581ef3eb2d25d0108b2d55eb5f1857R166), [link](https://github.com/ant-design/ant-design/pull/44000/files?diff=unified&w=0#diff-63d0bbc4ed8cb1745da77de552d75aa0fd581ef3eb2d25d0108b2d55eb5f1857R172), [link](https://github.com/ant-design/ant-design/pull/44000/files?diff=unified&w=0#diff-3a34112b09233455996fb191066f04b1c3fa01878e9400af4ca8a1dcf6c0106aR703), [link](https://github.com/ant-design/ant-design/pull/44000/files?diff=unified&w=0#diff-3a34112b09233455996fb191066f04b1c3fa01878e9400af4ca8a1dcf6c0106aR709))

### CODE REVIEW

File: components/input/style/index.ts [hunk 1]
```diff
@@ -700,11 +700,13 @@ const genGroupStyle: GenerateStyle<InputToken> = (token: InputToken) => {
         '&-lg': {
           [`${componentCls}-group-addon`]: {
             borderRadius: borderRadiusLG,
+            fontSize: token.fontSizeLG,
           },
         },
         '&-sm': {
           [`${componentCls}-group-addon`]: {
             borderRadius: borderRadiusSM,
+            fontSize: token.fontSizeSM,
```
Comment (MadCcc) [inline components/input/style/index.ts:709]: 小号的不一定要改，字体大小应该要和 input 一致

Comment (MuxinFeng) [reply components/input/style/index.ts:709]: 好的，addon 会默认给 14

**Comment by MadCcc** [comment]
image
看看这个是什么原因？

**Comment by MuxinFeng** [comment]
> image 看看这个是什么原因？

好嘞

**Comment by MuxinFeng** [comment]
> image 看看这个是什么原因？

看不出来有啥区别 😂，有啥排查思路吗~

**Comment by afc163** [comment]
addon 字体变化导致演示整体宽度变小的联动变化，符合预期。

**Review by afc163 — APPROVED** [review]

**Pull request merged**